### PR TITLE
Primitive project parsing

### PR DESCRIPTION
**Pull request by nickmessing**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  ],
+  "author": "Nick Messing <dot.nick.dot.messing@gmail.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/vueact/vueact-parse/issues"
+  },
+  "homepage": "https://github.com/vueact/vueact-parse#readme",
+  "devDependencies": {
+    "ava": "^0.21.0",
+    "eslint": "^4.3.0",
```
Comment (egoist) [inline package.json:28]: maybe use https://github.com/sindresorhus/xo directly

File: package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "eslint-config-rem": "^3.1.0",
+    "eslint-config-xo": "^0.18.2",
+    "eslint-plugin-react": "^7.1.0",
```
Comment (egoist) [inline package.json:31]: eslint-config-xo-react?

File: package.json (hunk 1 above)
Comment (nickmessing) [reply package.json:28]: @egoist, I need `no-await-in-loop` disabled

File: package.json (hunk 2 above)
Comment (nickmessing) [reply package.json:31]: @egoist, we don't actually need `react` rules, we will probably try to handle something that breaks `eslint-config-xo-react`, I just needed to get rid of `unused` error with `import React`

File: package.json (hunk 1 above)
Comment (egoist) [reply package.json:28]: you can still disable rules while using xo

Comment (nickmessing) [reply package.json:28]: Yep, I'll add `xo` and optimise it to be parallael instead of serial, I found out this rule is actually useful :+1:

**Pull request closed without merging**